packages/terra-form-field/CHANGELOG.md

```diff
@@ -18,6 +18,16 @@
 
 ## 4.20.3 - (April 29, 2022)
 
+* Changed
+  * Minor dependency version bump
+
+## 4.20.4 - (May 3, 2022)
+
+* Changed
+  * Minor dependency version bump
+
+## 4.20.3 - (April 29, 2022)
+
```

Comment:
These look to be duplicates as these versions already listed above.

Reply:
Removed here: 
https://github.com/cerner/terra-core/pull/3665/commits/551fb779b8a7b0b6187d726095ecad8bf1a7f39d